app/src/main/java/tech/ula/utils/AndroidUtility.kt

```diff
@@ -398,4 +399,54 @@ class AcraWrapper {
     fun putCustomString(key: String, value: String) {
         ACRA.getErrorReporter().putCustomData(key, value)
     }
+}
+
+class UsageUtility(private val prefs: SharedPreferences) {
+    private val numberOfTimesOpenedKey = "numberOfTimesOpened"
+    private val userGaveFeedbackKey = "userGaveFeedback"
+    private val dateTimeFirstOpenKey = "dateTimeFirstOpen"
+    private val millisecondsInThreeDays = 259200000L
+    private val minimumNumberOfOpensBeforeReviewRequest = 15
+
+    fun addToNumberOfTimesOpened(): Int {
```

Comment:
I like `incrementNumberOfTimesOpened` more personally, but that's nitpicky.

Reply:
I agree with that name 👍 I've changed it to `incrementNumberOfTimesOpened`